docs/developer-docs/latest/setup-deployment-guides/deployment/optional-software/caddy-proxy.md

```diff
@@ -52,24 +57,27 @@ http://api.example.com {
 
 #### Sub-Folder Unified
 
-This config is using a sub-folder that is dedicated to Strapi only. It will bind to port 80 HTTP and hosts the "frontend" files on `/var/www` like a normal web server, but proxies all Strapi requests on the `example.com/api` sub-path.
+This config is using a sub-folder that is dedicated to Strapi only. It will bind to port 80 HTTP and hosts the "frontend" files on `/var/www` like a normal web server, but proxies all Strapi requests on the `example.com/test` sub-path.
```

Comment:
```suggestion
This configuration is using a sub-folder that is dedicated to Strapi only. It will bind to port 80 HTTP and hosts the "front-end" files on `/var/www` like a normal web server, but proxies all Strapi requests on the `example.com/test` sub-path.
```